utilities/common.py

```diff
@@ -195,6 +197,7 @@ def common_parser(for_reco_highPU=False):
         parser.add_argument("--excludeFlow", action='store_true', help="Excludes underflow and overflow bins in main axes")
         parser.add_argument("--biasCalibration", type=str, default=None, choices=["binned","parameterized", "A", "M"], help="Adjust central value by calibration bias hist for simulation")
         parser.add_argument("--noSmearing", action='store_true', help="Disable resolution corrections")
+        parser.add_argument("--smearingUnc", action='store_true', help="add uncertainties from the resolution correction")
```

Comment:
I think we want this on by default

Reply:
I think probably you can just trigger it with --noSmearing rather than having a separate flag. Assuming it isn't super slow, I don't think we need it configurable to this level